spec/support/auth.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Auth
+  def create_user!
+    @user = Fabricate(:user)
+  end
+
+  def sign_in_user!
```

Comment:
Whats the syntax with ! at the end of method names?

Reply:
It's just convention but it often is used to say that a method is dangerous or have a side effect, which in this case is the user being logged in